propagator/opentelemetry-propagator-aws-xray/src/opentelemetry/propagators/aws/version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = "1.0.2"
+__version__ = "1.1.0.dev"
```

Comment:
Has this been bumped to 1.1.0.dev because these changes are thought as adding features right?

Reply:
Since we had no automation for these packages in the past, we currently have the version that's already been released -https://pypi.org/project/opentelemetry-propagator-aws-xray/1.0.2/

For packages that are released with workflow we use the vNext (minor + 1). 

So I'm just mimicking the same state/strategy as if it was released automatically:

- next version is assumed to be minor+1 (expecting new features and bot expecting patches)
- patch release should be done from the long-lived release branch. It should be possible to backport fixes to that branch when necessary.

Having said that, it's possible to release a patch from the main branch by updating the version on main to `1.0.3.dev` and running the release workflow - it'll prepare release `1.0.3` and send PR to bump version to `1.1.0.dev` - here's an example https://github.com/lmolkova/opentelemetry-python-contrib/actions/runs/11225953867.

Reply:
Thanks for the explanation